samples/apps/copilot-chat-app/scripts/Start-Frontend.ps1

```diff
@@ -0,0 +1,39 @@
+<#
+.SYNOPSIS
+Initializes and runs the Copilot Chat frontend.
+
+.PARAMETER ClientId
+The client (application) ID for your AAD app registration.
+
+.PARAMETER TenantId
+The tenant (directory) ID for your AAD app registration.
+If you are using a personal MSA account, enter 'msa' here.
+Defaults to the 'common' endpoint.
+#>
+
+#Requires -Version 6
```

Comment:
Is this a comment?  If so, why not uncomment it?

Reply:
This is like a pre-interpreter directive for PowerShell. Ignored by the script but interpreted by PowerShell itself before running.